integration-tests/tests/errors/index.spec.js

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright 2020 Splunk Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const path = require('path');
+
+module.exports = {
+  'DOM resource 4xx': async function(browser) {
+    browser.globals.clearReceivedSpans();
+    const url = browser.globals.getUrl('/errors/templates/resource-4xx.ejs');
+    await browser.url(url);
+
+    const errorSpan = await browser.globals.findSpan(s => s.name === 'eventListener.error');
+    await browser.assert.ok(!!errorSpan);
+
+    const tags = errorSpan.tags;
+    await browser.assert.ok(tags['location.href'].includes('/errors/templates/resource-4xx.ejs'), `Checking location.href: ${tags['location.href']}`);
+    await browser.assert.strictEqual(tags['splunk.rumVersion'], '0.0.13');
```

Comment:
Let's just have each test concentrate on the tags from its data source - location.href, splunk.rumVersion, and app can all be tested by an (e.g.) coretags test or something.

Reply:
makes sense